compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/controlcode_lowering.mlir

```diff
@@ -211,120 +220,20 @@ module attributes {hal.executable.target = #executable_target_amdaie_xclbin_fb}
+        amdaie.npu.half_dma_cpy_nd  %4(%5 [] [] [] bd_id = %bd_id channel = %channel_3) : !amdaie.logicalobjectfifo<memref<2048xi8>>
 // CHECK: amdaie.npu.write_bd {bd_id = 0 : ui32, buffer_length = 256 : ui32, buffer_offset = 0 : ui32, col = 0 : ui32, enable_packet = true, iteration_current = 0 : ui32, iteration_size = 0 : ui32, iteration_stride = 0 : ui32, lock_acq_enable = false, lock_acq_id = 0 : ui32, lock_acq_val = 0 : i32, lock_rel_id = 0 : ui32, lock_rel_val = 0 : i32, next_bd = 0 : ui32, out_of_order_id = 0 : ui32, packet_id = 0 : ui32, packet_type = 0 : ui32, paddings_after = array<i32>, paddings_before = array<i32>, row = 0 : ui32, sizes = array<i32: 4, 16, 4>, strides = array<i32: 16, 2, 1>, use_next_bd = false, valid_bd = true}
 // CHECK: amdaie.npu.address_patch {arg_idx = 0 : ui32, bd_id = 0 : ui32, col = 0 : ui32, offset = 0 : ui32}
 // CHECK: %[[TOKEN_0:.+]] = amdaie.npu.push_to_queue async {bd_id = 0 : ui32, channel = 0 : ui32, col = 0 : ui32, direction = 0 : i32, repeat_count = 2 : ui32, row = 0 : ui32}
 // CHECK: amdaie.npu.dma_wait(%[[TOKEN_0]] : !amdaie.async_token)
-        %6 = amdaie.npu.dma_cpy_nd async_target %4(%5[0, 0, 0, 0] [2, 4, 16, 16] [0, 64, 8, 1] bd_id = %bd_id, [] [] []) : target_type = !amdaie.logicalobjectfifo<memref<2048xi8>>
-        amdaie.npu.dma_wait(%6 : !amdaie.async_target_token)
+        amdaie.npu.half_dma_cpy_nd  %4(%0 [] [] [] channel = %channel) : !amdaie.logicalobjectfifo<memref<2048xi8, 1 : i32>, 2>
+        %6 = amdaie.npu.half_dma_cpy_nd async %4(%5 [0, 0, 0, 0] [2, 4, 16, 16] [0, 64, 8, 1] bd_id = %bd_id channel = %channel_3) : !amdaie.logicalobjectfifo<memref<2048xi8>>
+        amdaie.npu.dma_wait(%6 : !amdaie.async_token)
 // CHECK: amdaie.npu.write_bd {bd_id = 0 : ui32, buffer_length = 256 : ui32, buffer_offset = 0 : ui32, col = 0 : ui32, enable_packet = true, iteration_current = 0 : ui32, iteration_size = 2 : ui32, iteration_stride = 32 : ui32, lock_acq_enable = false, lock_acq_id = 0 : ui32, lock_acq_val = 0 : i32, lock_rel_id = 0 : ui32, lock_rel_val = 0 : i32, next_bd = 0 : ui32, out_of_order_id = 0 : ui32, packet_id = 0 : ui32, packet_type = 0 : ui32, paddings_after = array<i32>, paddings_before = array<i32>, row = 0 : ui32, sizes = array<i32: 4, 16, 4>, strides = array<i32: 16, 2, 1>, use_next_bd = false, valid_bd = true}
 // CHECK: amdaie.npu.address_patch {arg_idx = 0 : ui32, bd_id = 0 : ui32, col = 0 : ui32, offset = 288 : ui32}
 // CHECK: %[[TOKEN_0:.+]] = amdaie.npu.push_to_queue async {bd_id = 0 : ui32, channel = 0 : ui32, col = 0 : ui32, direction = 0 : i32, repeat_count = 2 : ui32, row = 0 : ui32}
-        %7 = amdaie.npu.dma_cpy_nd async_target %4(%5[0, 0, 32, 32] [2, 4, 16, 16] [128, 64, 8, 1] bd_id = %bd_id, [] [] []) : target_type = !amdaie.logicalobjectfifo<memref<2048xi8>>
-        amdaie.npu.dma_wait(%7 : !amdaie.async_target_token)
-        amdaie.end
-      }
-    }
-    return
-  }
-}
-
-// -----
-
-// CHECK-LABEL: @half_npu_dma_cpy_nd
```

Comment:
Not sure why you're removing the `half_npu_dma_cpy_nd` tests and keeping the `npu_dma_cpy_nd` ones? I would have expected it the other way around? And unless they are exactly duplicates, you could also keep both of them?